<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recon_configuration_page_file_size</class>
 <widget class="QWidget" name="recon_configuration_page_file_size">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget_filter_label">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableView {
/*selection-background-color: #87CEFF;*/
selection-background-color: #0069EB;
selection-color:white;
 }






QTableView::item:selected:!active {
    color:white;
    
   /*background-color:  #0069EB;*/


}




 QTableView::item:selected:!active {
                                 color:white;
                                 background-color:  #0069EB;
                                  }


                                 QTableView {
                                 selection-background-color: #0069D7;
                                    }




                                 QTableView::item:selected:!active {
                                 color:white;
                                 background-color:  #0069EB;
                                 }

QScrollBar{
		width:9px;
}</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Filter Label</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_add_label">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	margin: 1px;
     border-color: #7f7f7f;
     border-style: solid;
     border-radius: 1px;
     border-width: 1px;
      }
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #FFFFFF);
                border-color: #000000;
}

QPushButton:checked {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #FFFFFF);
                border-color: #484848;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_remove_label">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	margin: 1px;
     border-color: #7f7f7f;
     border-style: solid;
     border-radius: 1px;
     border-width: 1px;

      }
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #FFFFFF);
                border-color: #000000;
}

QPushButton:checked {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #FFFFFF);
                border-color: #484848;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_filter_details">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
/*selection-background-color: #87CEFF;*/
selection-background-color: #0069EB;
selection-color:white;
 }






QTableView::item:selected:!active {
    color:white;
    
   /*background-color:  #0069EB;*/


}




 QTableView::item:selected:!active {
                                 color:white;
                                 background-color:  #0069EB;
                                  }


                                 QTableView {
                                 selection-background-color: #0069D7;
                                    }




                                 QTableView::item:selected:!active {
                                 color:white;
                                 background-color:  #0069EB;
                                 }

QScrollBar{
		width:9px;
}</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Filter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size Unit</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
